tests/modules/dshbio/splitgff3/nextflow.config

```diff
@@ -3,7 +3,7 @@ process {
     publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
 
     withName: DSHBIO_SPLITGFF3 {
-        ext.suffix = '.'
+        ext.prefix = { "${meta.id}." }
```

Comment:
Maybe this should be kept just as `.`? I gues it's specifying an ourput directory?

Reply:
Nope. So it's actually the prefix for any output files but the `.` is required here to get for example to get `test.1.bed.gz` instead of `test1.bed.gz`. 

https://github.com/nf-core/modules/blob/1f8f86b79361fb9999b56680105f439df81d718d/tests/modules/dshbio/splitbed/test.yml#L7

Reply:
Ah understood :+1: